packages/pytest-simcore/src/pytest_simcore/docker_registry.py

```diff
@@ -36,7 +39,7 @@ def docker_registry(keep_docker_up: bool) -> str:
         )
 
         # Wait until we can connect
-        assert _wait_till_registry_is_responsive(url)
+        assert wait_till_registry_is_responsive(url)

```

Comment:
MINOR: why not having the code of this functio nhere with a classic

```python
  for attempt in Retrying(...)
    with attempt:
       ...
```

Reply:
as I said I don't like this retrying. it makes the whole thing too big. I prefer having a consistent way that works the same for both sync/async. this way if someone copy pastes the code it will work in async as well.

Reply:
ok. i did not read that before. makes sense.